src/dnx.clr/dnx.clr.cpp

```diff
@@ -23,15 +24,14 @@ extern "C" __declspec(dllexport) HRESULT __stdcall CallApplicationMain(PCALL_APP
     hr = bootstrapper->InitializeRuntime(data->runtimeDirectory, data->applicationBase, framework);
     if (SUCCEEDED(hr))
     {
-        dnx::utils::wait_for_debugger(data->argc, data->argv);
+        dnx::utils::wait_for_debugger(data->argc, data->argv, L"--debug");
 
         g_clrBootstrapper = NULL;
         data->exitcode = bootstrapper->CallApplicationMain(data->argc, data->argv);
     }
     else
     {
-        printf_s("Failed to initialize runtime (%x)", hr);
-        data->exitcode = hr;
```

Comment:
We only return exitcode if hr is S_OK (https://github.com/aspnet/dnx/blob/dev/src/dnx/pal.win32.cpp#L146) so it is not needed here because hr is not S_OK.